### PR TITLE
Replace Travis CI with Github Actions

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: test
+
+on: [push, pull_request]
+
+jobs:
+  ansible-molecule:
+    runs-on: macos-10.15
```
Comment (francisco-core) [inline .github/workflows/test.yml:7]: Why MacOS?

**Review by francisco-core — COMMENTED** [review]
Preliminary review (still need to read up on github actions). Is this supposed to replace the current travis workflow? If so:

1. Why do we need this?
2. Shouldn't we also remove the travis-ci files

**Comment by anadahz** [comment]
> Preliminary review (still need to read up on github actions). Is this supposed to replace the current travis workflow? If so:
> 
>     1. Why do we need this?

Yes, Travis CI is no longer providing CI minutes for open source projects.

For more info, please read:
https://www.jeffgeerling.com/blog/2020/travis-cis-new-pricing-plan-threw-wrench-my-open-source-works

>     2. Shouldn't we also remove the travis-ci files

Yes, we should do.
I have changed the title to reflect that this is a WIP PR.

Thanks for looking into it.

**Comment by anadahz** [comment]
Useful to this PR:
https://github.com/jonashackt/vagrant-github-actions

File: .github/workflows/test.yml (hunk 1 above)
Comment (anadahz) [reply .github/workflows/test.yml:7]: Currently Vagrant support for nested virtualization works only on GitHub Action MacOS environment:
https://github.com/actions/virtual-environments/issues/433

Comment (francisco-core) [reply .github/workflows/test.yml:7]: How weird

**Comment by francisco-core** [comment]
Things seem to be failing with very generic error messages. Is this on purpose?

**Comment by anadahz** [comment]
> Things seem to be failing with very generic error messages. Is this on purpose?

Which things specifically?

**Comment by francisco-core** [comment]
if you click on the thing bellow `test / ansible-molecule (mailcow) (push) Failing after 6m — ansible-molecule (mailcow)`:

> Testing ansible playbook `deploy-mailcow`
> * Error: 

And nothing more

**Comment by anadahz** [comment]
> if you click on the thing bellow `test / ansible-molecule (mailcow) (push) Failing after 6m — ansible-molecule (mailcow)`:
> 
> > Testing ansible playbook `deploy-mailcow`
> > 
> > * Error:
> 
> And nothing more

You should be able to see the complete steps and errors.
For instance here: https://github.com/PrivacyLx/devops/pull/68/checks?check_run_id=2858778386

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+name: test
+
+on: [push, pull_request]
```
Comment (francisco-core) [inline .github/workflows/test.yml:3]: podemos fazer apenas tests em push? Quando é pull request ele faz em duplicado e fica um pouco confuso de ver o que faz o quê.

**Review by francisco-core — APPROVED** [review]
The matterbridge is failing apparently for network issues. For me we can merge this and then address that in case it persists. Thanks a lot for working on this.

Approved.

File: .github/workflows/test.yml (hunk 2 above)
Comment (anadahz) [reply .github/workflows/test.yml]: Please see: https://github.com/PrivacyLx/devops/pull/68/commits/91438cb6ea3bea42243dfefaf764b0d982497d6d

**Comment by anadahz** [comment]
@francisco-core thank you for reviewing this PR.

**Pull request merged**